### ISSUE TITLE
HTTP Error 429: Too Many Requests

### ISSUE THREAD

**Issue by rbkhb** [marquisvictor/Modified-GetOldTweets3#5]
Starting last week, this error occurs after 14300 tweets are scraped: 

> Saved 14300An error occured during an HTTP request: HTTP Error 429: Too Many Requests

Anyone have a fix?

**Comment by marquisvictor** [comment]
Did you fix this already?

**Comment by Ledgrev** [comment]
Hello @marquisvictor, I just wanted to let you know that I am experiencing the same problem, so I think @rbkhb might still have the same issue as well.

**Comment by sagar-aps** [comment]
Same problem, I get it with 14400 tweets.

**Comment by sagar-aps** [comment]
I did some testing to try and get around the problem. Here is what I found. The number 14400, 14200 is irrelevant, the error comes after the server is queried continuously for a certain amount of time. Wait period is also important. I got the error even while running commands fetching <5000 tweets if I had just run another command fetching 6000 tweets before the second.

My workaround was to make a batch file with the TIMEOUT /T command and to divide up tweet fetching. It still fails if the tweets targeted exceed 14400 (after x seconds of continuous querying)

Usually, 429 errors will specify a timeout. If this is added in the python script as a pause, the program may be able to download continously.

[Error 429 file 1.txt](https://github.com/marquisvictor/Optimized-Modified-GetOldTweets3-OMGOT/files/4895844/Error.429.file.1.txt)
[Error 429 file 2.txt](https://github.com/marquisvictor/Optimized-Modified-GetOldTweets3-OMGOT/files/4895845/Error.429.file.2.txt)

[download_by_month - SecondPass.bat.txt](https://github.com/marquisvictor/Optimized-Modified-GetOldTweets3-OMGOT/files/4895848/download_by_month.-.SecondPass.bat.txt)
